### PR TITLE
Code style, remove typo, double ;; from code where it makes sense to do so

### PR DESCRIPTION
**Pull request by jusfreeman**
Code style, remove typo, double ;; from code where it makes sense to do so.

As noted whilst reviewing https://github.com/civicrm/civicrm-core/pull/15937

Agileware Ref: CIVICRM-1383

### CODE REVIEW

File: js/jquery/jquery.crmProfileSelector.js [hunk 1]
```diff
@@ -13,7 +13,7 @@
    * <input type="text" value="{$profileId}" class="crm-profile-selector" />
    * ...
    * cj('.crm-profile-selector').crmProfileSelector({
-   *   groupTypeFilter: "Contact,Individual,Activity;;ActivityType:7",
```
Comment (colemanw) [inline js/jquery/jquery.crmProfileSelector.js]: Are you sure about this one? It's neither javascript code nor php code. It's a data serialization notation.

Comment (jusfreeman) [reply js/jquery/jquery.crmProfileSelector.js]: @colemanw it's also just a code comment - not executable code

Comment (colemanw) [reply js/jquery/jquery.crmProfileSelector.js]: Agree it's a comment, but according to [this code documentation](https://github.com/civicrm/civicrm-core/blob/5.19/js/model/crm.uf.js#L46) it's actually correct using the double semicolon.

Comment (jusfreeman) [reply js/jquery/jquery.crmProfileSelector.js]: @colemanw reverted that change

**Comment by yashodha** [comment]
@jusfreeman looks good. Can you please squash your commits?

**Comment by jusfreeman** [comment]
@yashodha done

**Comment by yashodha** [comment]
@jusfreeman Thanks! Will merge once tests pass.

**Pull request merged**